usr/local/openresty/nginx/templates/pagespeed.conf.template

```diff
@@ -1,5 +1,5 @@
 # Enable PageSpeed
-pagespeed on;
+pagespeed off;
```

Comment:
I think this should be `standby` or `unplugged`
See https://www.modpagespeed.com/doc/configuration#on_off

Reply:
https://github.com/CruGlobal/cruorg_proxy/pull/34